.eslintignore

```diff
@@ -0,0 +1,2 @@
+dist/**
```

Comment:
Why are you not linting "dist"?  What kinds of failures result if you lint that?

Reply:
Basically the files in this directory are copied from the `src` directory itself which we already lint. Moreover, this directory is not committed to the repository so we don't necessarily need to do it. However, if you think that we should still lint it for some reason, please let me know.

Reply:
No, I figured out what you were doing and why later on, it's fine.  As long as the code you write lints, I'm happy.